src/app/services/age-verification/age-verification.service.spec.ts

```diff
@@ -33,4 +33,20 @@ describe('AgeVerificationService', () => {
 
     expect(result).toContain('is a minor');
   });
+
+  it('should returns false when age is lowerthen legal age', () => {
+    const userAge = 17;
+
+    const result = service.isLegalAge(userAge);
+
+    expect(result).toBeFalsy();
+  });
+
+  it('should return true when age is equal or greater than legal age', () => {
```

Comment:
I will suggest to apply KISS in this phrase, change `equal or greater than` to `gte`(greater than or equal).